crates/genesis/src/lib.rs

```diff
@@ -64,14 +66,14 @@ pub struct Genesis {
     // should NOT be set in a real genesis file, but are included here for compatibility with
     // consensus tests, which have genesis files with these fields populated.
     /// The genesis header base fee
-    #[serde(skip_serializing_if = "Option::is_none", with = "u64_hex_or_decimal_opt")]
-    pub base_fee_per_gas: Option<u64>,
+    #[serde(skip_serializing_if = "Option::is_none", with = "u128_hex_or_decimal_opt")]
+    pub base_fee_per_gas: Option<u128>,
     /// The genesis header excess blob gas
-    #[serde(skip_serializing_if = "Option::is_none", with = "u64_hex_or_decimal_opt")]
-    pub excess_blob_gas: Option<u64>,
+    #[serde(skip_serializing_if = "Option::is_none", with = "u128_hex_or_decimal_opt")]
+    pub excess_blob_gas: Option<u128>,
     /// The genesis header blob gas used
-    #[serde(skip_serializing_if = "Option::is_none", with = "u64_hex_or_decimal_opt")]
-    pub blob_gas_used: Option<u64>,
+    #[serde(skip_serializing_if = "Option::is_none", with = "u128_hex_or_decimal_opt")]
+    pub blob_gas_used: Option<u128>,
```

Comment:
Those need `#[serde(default)]` ref https://github.com/alloy-rs/alloy/pull/370

Reply:
good catch,
always forget that missing key is not handled by option